mne/channels/channels.py

```diff
@@ -260,19 +261,19 @@ def set_eeg_reference(self, ref_channels=None, verbose=None):
         Average reference:
             A new virtual reference electrode is created by averaging the
             current EEG signal. Make sure that all bad EEG channels are
-            properly marked and set ``ref_channels=None``.
+            properly marked and set ``ref_channels='average'``.
 
         A single electrode:
-            Set ``ref_channels`` to the name of the channel that will act as
-            the new reference.
+            Set ``ref_channels`` to a list containing the name of the channel
+            that will act as the new reference, for example
+            ``ref_channels=['Cz']`.
 
-        The mean of multiple electrodes:
-            A new virtual reference electrode is created by computing the
-            average of the current EEG signal recorded from two or more
-            selected channels. Set ``ref_channels`` to a list of channel names,
-            indicating which channels to use. For example, to apply an average
-            mastoid reference, when using the 10-20 naming scheme, set
-            ``ref_channels=['M1', 'M2']``.
+        The mean of multiple electrodes: A new virtual reference electrode
```

Comment:
Argh, I tried PyCharm's "Fill Paragraph" function but obviously it didn't work. Can PyCharm automatically re-format comments or do I really have to do this manually?